_dprhtml/index.html

```diff
@@ -42,7 +42,12 @@
         "DPR_BottomPane",
         "__otherDialogsViewModel",
         "w",
-        "Microsoft"
+        "Microsoft",
+        "Nanobar",
+        "$",
+        "jQuery",
+        "Popper",
+        "DPR_DataLoader"
```

Comment:
Add comma at the end, so next time you add new entries, a red line does not show up in the diff.